device-data/out-of-range.md

```diff
@@ -1,5 +1,94 @@
 <!-- non-generated document! all areas editable -->
 
-Coming soon!
+ ## Out of Range Value
+ Blood glucose meters, ketone meters and Continuous Glucose Monitors have a range within which they
+ can measure levels. When a reading falls outside of this range, we need a mechanism to show that we
+ have received a reading, but that it is out of range.
 
-<!-- TODO: document on out-of-range values, relevant especially to cbg, smbg, and bloodKetone records -->
+ This is done by creating a `cbg`, `smbg` or `bloodKetone` reading with a `value` that is 1 unit
+ over (for a high reading) or 1 unit below (for a low reading) the range threshold, and then annotating
+ the reading.
+
+For example, for a low CGM reading for a device that reads in mg/dL with a low threshold of 40 mg/dL:
+```json
+{
+  "type": "cbg",
+  "units": "mg/dL",
+  "value": 39,
+  "clockDriftOffset": 0,
+  "conversionOffset": 0,
+  "deviceId": "DevId0987654321",
+  "deviceTime": "2016-06-13T19:05:43",
+  "guid": "cf64b6f6-54af-4e8b-a432-08d53c2484d4",
+  "id": "da34f6f278984ae385c91cac6b91eff7",
+  "time": "2016-06-14T02:05:43.959Z",
+  "timezoneOffset": -420,
+  "uploadId": "SampleUploadId",
+  "annotations": [{
+    "code": "bg/out-of-range",
+    "value": "low",
+    "threshold": 40,
+  }]
+}
+```
+
+Unlike all of the other data model types, the `out-of-range` annotation does not currently include a field for `units`.
+Because of this, when annotating `cbg` or `smbg` readings, the `threshold` should **always** be in `mg/dL`,
+even when the device takes readings in `mmol/L`.
```

Comment:
I would make a historical note here that the reason for this is *because* so many devices store in mg/dL even if they display in mmol/L (and overall I prefer the terminology of what devices "store" vs. "display" over "read," which is a bit ambiguous). And perhaps an additional note below that your mmol/L example is manufactured (has not happened yet in our experience).